src/state/slices/opportunitiesSlice/selectors/stakingSelectors.ts

```diff
@@ -403,10 +387,11 @@ export const selectAggregatedEarnUserStakingOpportunities = createDeepEqualOutpu
             .div(bn(10).pow(asset?.precision ?? underlyingAsset?.precision ?? 1))
             .toFixed(),
           cryptoAmountBaseUnit: opportunity.stakedAmountCryptoBaseUnit,
-          fiatAmount: bnOrZero(opportunity.stakedAmountCryptoBaseUnit)
-            .times(marketData[asset?.assetId ?? underlyingAsset?.assetId ?? '']?.price ?? '0')
-            .div(bn(10).pow(asset?.precision ?? underlyingAsset?.precision ?? 1))
-            .toString(),
+          fiatAmount: makeOpportunityTotalFiatBalance({
+            opportunity,
+            marketData,
+            assets,
+          }).toString(),
```

Comment:
Fiat amount now uses the exact same logic as `<EarnBalance />` and includes the actual total amount for the opportunity

Reply:
On second thought, probably not: rewards can exist in multiple denoms, and total fiat amount should really equal to the fiat amount of staked amount + rewards + undelegations, which it does